config/rbac/hive_regex_role.yaml

```diff
@@ -0,0 +1,16 @@
+# hive-regex is a role intended for hive administrators who need to be able to modify hive install log monitor configuration.
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: hive-regex
+rules:
+- apiGroups:
+  - ""
+  resources:
+  - "configmaps"
+  resourceNames:
+  - "additional-install-log-regexes"
+  verbs:
+  - get
+  - patch
+  - update
```

Comment:
If this is managed in app-interface, I don't think we need patch/update here, and we already have get. Feels safe to drop the role changes, wdyt?

Reply:
I agree. I'll remove them.